client/containers/Pub/SpubHeader/spubHeader.scss

```diff
@@ -111,3 +111,15 @@ $title-elements-spacing: 15px;
 		max-width: $spub-configuration-width;
 	}
 }
+
+.small-header-button-component {
```

Comment:
Don't forget that these need to be scoped to the `.spub-header-component` selector or they're going to apply everywhere in the application!